app/Http/Transformers/PhoenixGalleryTransformer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Post;
+use League\Fractal\TransformerAbstract;
+
+class PhoenixGalleryTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param \Rogue\Models\Photo $photo
+     * @return array
+     */
+    public function transform(Post $post)
+    {
+        $signup = $post->signup;
+        $result = [
+            'id' => $post->postable_id,
+            'status' => $post->status,
+            'caption' => $post->content->caption,
+            'uri' => $post->content->file_url,
```

Comment:
@sbsmith86 this is what is in there:
```
      uri: "https://www.dosomething.org/api/v1/reportback-items/658",
      media: {
        uri: "https://www.dosomething.org/sites/default/files/styles/480x480/public/reportbacks/362/image42.jpeg",
        type: "image"
      }
```
they are two different URLs and I wasn't sure what they were used for so I put it in there twice. @chloealee do you know if you use both of these?